src/routes/+layout.svelte

```diff
@@ -1,22 +1,17 @@
 <script lang="ts">
 	import { dev } from "$app/environment";
 	import { page } from "$app/stores";
-
```

Comment:
Why did you remove all the blank lines in this file? Please add them back. all formatting in this project should be done according to the rules defined in the prettier config.

Reply:
ok